juntagrico/admins/member_admin.py

```diff
@@ -8,13 +9,18 @@
 
 class MemberAdmin(BaseAdmin):
     form = MemberAdminForm
-    list_display = ['email', 'first_name', 'last_name']
+    list_display = ['email', 'first_name', 'last_name', 'active']
     search_fields = ['first_name', 'last_name', 'email']
-    # raw_id_fields = ['subscription']
     exclude = ['future_subscription', 'subscription', 'old_subscriptions']
     readonly_fields = ['user']
     actions = ['impersonate_job']
 
+    def active(self, instance):
+        return not instance.inactive
```

Comment:
sollte das nicht eher ins member object rein? vorallem wenn wir zu einem späteren zeitpunkt nicht mehr mit flags sondern mit daten arbeiten?

Reply:
Offenbar gibt es keine schöne Lösung ein Property als Bool anzeigen zu lassen:
https://stackoverflow.com/questions/12842095/how-to-display-a-boolean-property-in-the-django-admin
Mit der Lösung hier, müsste die Methode dann sowieso hierhin auch wenns im Model wäre:
https://stackoverflow.com/a/51052597/1656781

Reply:
ich habe das 1:1 ins member kopiert (ok nicht 1:1 ich habe das instance rausgenommen) und es macht daselbe

Reply:
Hast aber keine property daraus gemacht. Ist auch eine Option.

Reply:
dann hat man weniger code duplication

Reply:
Sofern dus nie zu einem property machen willst geht's.